### PR TITLE
Replace context tag constant for marathi language

### PR DESCRIPTION
**Pull request by ankur09011**
## JIRA Ticket Number

JIRA TICKET: ML-2466

## Description of change
Replace context tag constant for marathi language. New values will be all constants for hindi transalted to marathi

## Checklist (OPTIONAL):

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: ner_v1/detectors/textual/name/lang_constants.py [hunk 1]
```diff
@@ -240,7 +240,13 @@
                           "your full name",
                           "what is your name",
                           "is your name"],
-                   'mr': ['नाव', 'नाव', '*नाव*', '*नाव*'],
+                   'mr': ['आपले नाव सांगायचे विसरलात', 'आपले पूर्ण नाव सांगून मला मदत करा', 'आपले पूर्ण नाव सांगून मला मदत करा',
```
Comment (haptik-deployment) [inline ner_v1/detectors/textual/name/lang_constants.py:243]: E501 line too long (128 > 119 characters)

File: ner_v1/detectors/textual/name/lang_constants.py [hunk 2]
```diff
@@ -240,7 +240,13 @@
...
+                          'मी आपण आपले नाव सांगायचे विसरलात वाटते', 'आपले नाव सिंह', 'आपले नाव माहीत', 'मला आपले नाव सांगा',
```
Comment (haptik-deployment) [inline ner_v1/detectors/textual/name/lang_constants.py:244]: E501 line too long (124 > 119 characters)

File: ner_v1/detectors/textual/name/lang_constants.py [hunk 3]
```diff
@@ -240,7 +240,13 @@
...
+                          'आपले नाव सांगा', 'आपण काय म्हणू नये', 'आम्हाला आपले नाव सांगू', 'आपण म्हणतात', 'आपण पत्ता',
+                          'आपले नाव', 'तुमचे पुर्ण नाव', 'तुमचे नाव आहे', 'आपले नाव', 'आपले नांव लिहा', 'आपले नाव प्रविष्ट करा',
```
Comment (haptik-deployment) [inline ner_v1/detectors/textual/name/lang_constants.py:246]: E501 line too long (128 > 119 characters)

File: ner_v1/detectors/textual/name/lang_constants.py [hunk 4]
```diff
@@ -240,7 +240,13 @@
...
+                          'कृपया आपले पूर्ण नाव मला सांग.', 'तुमचे पूर्ण नाव काय आहे?', 'तिच्या स्वत: च्या नाव सांगू मला मदत',
```
Comment (haptik-deployment) [inline ner_v1/detectors/textual/name/lang_constants.py:247]: E501 line too long (126 > 119 characters)

File: ner_v1/detectors/textual/name/lang_constants.py [hunk 5]
```diff
@@ -240,7 +240,13 @@
...
+                          'आपले नाव द्या', 'आम्हाला आपले नाव सांगू', 'पुन्हा आपले पूर्ण नाव प्रविष्ट करा', 'तुमचे नाव आहे',
```
Comment (haptik-deployment) [inline ner_v1/detectors/textual/name/lang_constants.py:248]: E501 line too long (123 > 119 characters)

**Comment by haptik-deployment** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Review by hemantgandhi90 — DISMISSED** [review]

File: ner_v1/detectors/textual/name/lang_constants.py [hunk 6]
```diff
@@ -1,22 +1,15 @@
...
-           'फटकेबाजीत लक्ष्', 'ढेकूळ'],
+    'mr': ['आपले नाव सांगायचे विसरलात', 'आपले पूर्ण नाव सांगून मला मदत करा',
+           'आपले पूर्ण नाव सांगून मला मदत करा', 'आपण आपल्या नावाचा उल्लेख करणे विसरलात असे दिसते',
+           'आपले नाव सामायिक करा', 'आपले नाव जाणून घ्या', 'मला आपले नाव सांगा', 'आपले नाव सांगा',
+           'मी तुम्हाला काय बोलावे?', 'आपले नाव सांगा', 'तुम्हाला बोलवल', 'तुम्हाला संबोधित केले',
+           'आपले नाव', 'तुमचे पुर्ण नाव', 'तुमचे नाव काय आहे', 'आपले नाव', 'आपले नांव लिहा',
+           'आपले नाव प्रविष्ट करा', 'कृपया आपले पूर्ण नाव सांगून मला मदत करा', 'तुमचे पूर्ण नाव काय आहे?',
+           'मला तुमचे नाव सांगण्यास मदत करा', 'आपले नाव द्या', 'आपले नाव सांगा',
+           'पुन्हा आपले पूर्ण नाव प्रविष्ट करा', 'तुमचे नाव काय आहे', 'स्वत: चे नाव', 'आपले नाव भरा',
+           'तुझं नाव काय आहे']
```
Comment (haptik-deployment) [inline ner_v1/detectors/textual/name/lang_constants.py:11]: E203 whitespace before ','

File: ner_v1/detectors/textual/name/lang_constants.py [hunk 7]
```diff
@@ -1,22 +1,15 @@
...
+,
```
Comment (haptik-deployment) [inline ner_v1/detectors/textual/name/lang_constants.py:12]: E122 continuation line missing indentation or outdented

**Comment by haptik-deployment** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
:+1: No lint errors found.

**Pull request merged**